public/assets/sass/carousel.scss

```diff
@@ -0,0 +1,95 @@
+* {box-sizing:border-box}
```

Comment:
it's not just applied to the carousel - also to the div of dots right underneath. removing this line and applying it to just slideshow (or even applying it to both slideshow and dot) creates some weird width css messup that i can't figure out which is why its applied to everything lols

Reply:
alright, sounds good for now! I think we should try to avoid applying styles to all elements like this in general, but since we're so early on it's good for now lol 😅